### ISSUE TITLE
Add shuttle configuration output for debugging

### ISSUE THREAD

**Issue by Crevil** [lunarway/shuttle#21]
To aid troubleshooting issues it would be great to be able to get shuttle version, current plan and version etc. output.

Something like `go env` in the go tool chain or `npm version` for Node.js.

```
$ go env
GOARCH="amd64"
GOBIN="/Users/bjornsorensen/go/bin"
GOCACHE="/Users/bjornsorensen/Library/Caches/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="darwin"
GOOS="darwin"
GOPATH="/Users/bjornsorensen/go"
...
```

```$ npm version
{ npm: '6.4.1',
  ares: '1.14.0',
  cldr: '33.1',
  http_parser: '2.8.0',
  icu: '62.1',
  modules: '64',
  napi: '3',
  nghttp2: '1.33.0',
  node: '10.11.0',
  openssl: '1.1.0i',
  tz: '2018e',
  unicode: '11.0',
  uv: '1.23.0',
  v8: '6.8.275.32-node.28',
  zlib: '1.2.11' }
```

When looking into an issue we can derive much information of such reports. I'm thinking something like

```
$ shuttle config
version 0.5.2

Environment:
LW_PATH=/user/...

Plan:
git://git@bitbucket.org:LunarWay/lw-shuttle-go-plan.git c331cbb3b71970dc6a3afabcd9320fb070bb719d
```

Maybe even let plans specify specific information related to the plan, ie. `go env` for Go plans etc.

**Comment by michaelbui99** [comment]
Hello

in the example: 
```
Environment:
LW_PATH=/usr/...
```
What information is to be displayed for Environment? Is it the vars from the shuttle.yaml file?

**Comment by Crevil** [comment]
I think expect the initial idea was environment variables. It is sort of a "catch all" idea, but also with the risc of disclosing sensitive information, so maybe a first iteration would be to skip that part.

**Comment by michaelbui99** [comment]
Could maybe also do something like: 

```
$ shuttle config --exclude-env-vars ENV_VAR_WITH_SENSITIVE_INFORMATION1  ENV_VAR_WITH_SENSITIVE_INFORMATION2 ...
```

That would give the user some control over what is being displayed

**Comment by Crevil** [comment]
Closed in #133 and released in 0.16.0
